test/integration/acme/orders_controller_test.go

```diff
@@ -282,7 +287,7 @@ func TestAcmeOrdersController(t *testing.T) {
 			return false, nil
 		}
 		return true, nil
-	})
+	}, ctx.Done())
```

Comment:
Ah, I see. We don't use `timeoutCtx` here, we just wait for success or for the test-wide timeout.